LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -188,7 +189,15 @@ public bool HasPassword(User user)
         /// <inheritdoc />
         public Task ChangePassword(User user, string newPassword)
         {
-            throw new NotImplementedException();
+            if (!LdapPlugin.Instance.Configuration.AllowPassChange)
+            {
+                throw new NotImplementedException();
+            }
+
+            var ldapUser = LocateLdapUser(user.Username) ?? throw new LdapException("No users found in LDAP Query");
+            using var ldapClient = ConnectToLdap() ?? throw new AuthenticationException("Failed to Connect or Bind to server");
+            ldapClient.Modify(ldapUser.Dn, new LdapModification(LdapModification.Replace, new LdapAttribute("userPassword", newPassword)));
```

Comment:
`userPassword` should be configurable

Reply:
fixed this and noticed some missing bits at the bottom of configPage.html